src/Exceptions/CannotRegisterIconSet.php

```diff
@@ -15,7 +15,7 @@ public static function pathNotDefined(string $set): self
 
     public static function pathOrDiskNotDefined(string $set): self
     {
-        return new static("The options for the \"$set\" set don't have a path or disk defined.");
+        return new static("The options for the \"$set\" need provide either a path or disk option.");
```

Comment:
"The options for the \"$set\" need to be provided either a path or disk option."